index.js

```diff
@@ -298,16 +298,16 @@ function diffArguments(thisType, otherType, thisField, otherField, options) {
     let newArgsMap = getArgumentMap(otherField.args);
     oldArgsMap.forEach(function(value, key) {
         if (!newArgsMap.has(key)) {
-            const description = format('Argument missing from {0} : `"{1} : {2}"` on field {3}.{4}.', options.labelForThis, key, value, thisType, thisField.name);
+            const description = format('Argument missing from {0}: {1}.{2}({3}: {4}).', options.labelForThis, thisType, thisField.name, key, value);
```

Comment:
Let's add some backticks around the value: Argument missing from {0}: `{1}.{2}({3}: {4})`.